airflow/auth/managers/base_auth_manager.py

```diff
@@ -90,18 +91,22 @@ def get_user_display_name(self) -> str:
         """Return the user's display name associated to the user in session."""
         return self.get_user_name()
 
-    @abstractmethod
+    @deprecated(
```

Comment:
Hmm, feels weird to deprecate this but `get_user_name` and `get_user_display_name` aren't? I know you are targeting abstactmethods, but still.

Reply:
That was one of the reason. The other reason is, it is still unclear to me what the new UI needs. Are we thinking displaying user info on the UI? Are we thinking having a link towards a user profile page like the legacy UI? Maybe that's a question for @bbovenzi